locales/locales.php

```diff
@@ -687,6 +687,15 @@ public function __construct() {
 		$es_co->slug = 'es-co';
 		$es_co->google_code = 'es';
 		$es_co->facebook_locale = 'es_CO';
+		
+		$es_CR = new GP_Locale();
+		$es_CR->english_name = 'Spanish (Costa Rica)';
+		$es_CR->native_name = 'Español';
+		$es_CR->lang_code_iso_639_1 = 'es';
+		$es_CR->lang_code_iso_639_3 = 'spa';
```

Comment:
In the other locales that have both _2 & _3 and are the same we add both so I'd suggest adding both in this case as well.